src/components/rating/components/CdrStar00.vue

```diff
@@ -1,17 +1,10 @@
 <script setup lang="ts">
 import mapClasses from '../../../utils/mapClasses';
-import validateProp from "../../../utils/propValidator";
+import sizeProps from '../../../props/size';
 import { useStar } from '../composables/useStar';
 
 const props = defineProps({
-  size: {
-      type: String,
-      default: 'medium',
-      validator: (value: string) => validateProp(
-        value,
-        ['small', 'medium', 'large'],
-      ),
-    },
+  size: sizeProps,
```

Comment:
An example of a component that was able to use the common `size` prop util but wasn't